TestsExample/src/Test1032.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Alert, Button, Switch, Text, View } from 'react-native';
+import { NavigationContainer, ParamListBase } from '@react-navigation/native';
+import { createNativeStackNavigator, NativeStackNavigationProp } from 'react-native-screens/native-stack';
+
+const Stack = createNativeStackNavigator();
+
+export default function App() {
+    const [gestureEnabled, setGestureEnable] = React.useState(false);
+    return (
+        <View style={{ flex: 1, paddingBottom: 200 }}>
+            <NavigationContainer>
+                <Stack.Navigator screenOptions={{ gestureEnabled }}>
+                    <Stack.Screen name="Top" component={First} options={{ replaceAnimation: 'push' }} />
```

Comment:
`options={{ replaceAnimation: 'push' }}` is this option needed for something?

Reply:
No. Actually it is a leftover